test/function_test.py

```diff
@@ -1159,12 +1150,7 @@ def f():
     (tmp_path / "mod.py").touch()  # some file
     entrypoint_file.write_text(src)
 
-    monkeypatch.delenv("MODAL_AUTOMOUNT")
-    if config_automount is not None:
-        env = {**os.environ, "MODAL_AUTOMOUNT": config_automount}
-    else:
-        env = {**os.environ}
-    output = deploy_app_externally(servicer, credentials, str(entrypoint_file), env=env)
+    output = deploy_app_externally(servicer, credentials, str(entrypoint_file), env={**os.environ})
```

Comment:
one cool thing is that `deploy_app_externally` might not be needed after this change (lets do that separately though) - iirc it's mostly in place because we needed a clean separate interpreter to not have tests interact through automount side effects when they import modules